src/Parser/Mage/Fire/Modules/Features/HeatingUp.js

```diff
@@ -119,19 +119,21 @@ class HeatingUp extends Analyzer {
   }
 
   suggestions(when) {
-		when(this.fireBlastUtilSuggestionThresholds)
+		when(this.fireBlastMissedPercent).isLessThan(this.fireBlastUtilSuggestionThresholds.isLessThan.minor)
```

Comment:
It looks like you undid the suggestion threshold updates by accident...

Reply:
oh ... oops .... wonder how that happened